.gitignore

```
@@ -28,6 +28,8 @@ __pycache__
 .hg/
 .svn/
 .idea/
+.project
+.settings/
```

```suggestion
.project
.settings/
```
Let's keep this PR focused on one thing